assets/js/plugin-check-admin.js

```diff
@@ -0,0 +1,43 @@
+( function ( data ) {
+	const checkItButton = document.getElementById( 'pc_check_it' );
+	const pluginsList   = document.getElementById( 'pc_plugins' );
```

Comment:
For both of these, it's always a good practice to check whether those elements are actually present on the page. Of course we can assume they are, but there's always a chance that something goes wrong on the page (e.g. due to another plugin), and then we don't want to be causing a JS error.

I think if one of these returns `null`, we should return early. Maybe we can also do a quick `console.error` to inform that the element is missing.